apps/src/templates/progress/DetailProgressTable.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { PropTypes } from 'react';
+import ProgressLesson from './ProgressLesson';
+import i18n from '@cdo/locale';
+import { levelType } from './progressTypes';
+
+/**
+ * A component that shows progress in a course with more detail than the summary
+ * view
+ */
+const DetailProgressTable = React.createClass({
+  propTypes: {
+    lessonNames: PropTypes.arrayOf(PropTypes.string).isRequired,
+    levelsByLesson: PropTypes.arrayOf(
+      PropTypes.arrayOf(levelType)
+    ).isRequired,
+  },
+
+  render() {
+    const { lessonNames, levelsByLesson } = this.props;
+    return (
+      <div>
+        {lessonNames.map((lessonName, index) => (
+          <ProgressLesson
+            key={index}
+            title={i18n.lessonN({lessonNumber: index + 1}) + ": " + lessonName}
```

Comment:
We should probably include the concatenation here in the i18n too (e.g. `i18n.lessonNumAndName({lessonNumber: index + 1, lessonName})`) in case RTL languages expect to display `lessonName :lessonNumber` in the future.